uid: add get_eeprom_serial

lexszero opened the pull request:
Читает зашитый в EEPROM MAC-адрес и отдает его в виде строчки
```
>>> uid.get_eeprom_serial()
'd88039a45981'
```

lexszero committed:
uid: add get_eeprom_serial

evgeny-boger commented on wb_common/uid.py:
```diff
@@ -27,6 +27,19 @@ def get_mmc_serial():
+    if num == 0:
+        eeprom = "4-0057"
+    elif num == 1:
+        eeprom = "5-0057"
```
Мне не нравится, что здесь захардкожены адреса EEPROM и номера шин.

Предлагаю поменять на get_iface_mac() с проверкой на то, что он не рандомный и не присвоен из юзерспейса , как в wb-utils, а логику оставить в u-boot

lexszero committed:
uid: replace get_eeprom_serial with get_eth_mac

Retrieve MAC from local-mac-address DT attribute which is set by U-Boot
instead of reading EEPROM directly.

lexszero committed:
uid: check for dummy mac in get_eth_mac()
